Formula/o/openjazz.rb

```diff
@@ -37,26 +36,12 @@ class Openjazz < Formula
     sha256 "ed025415c0bc5ebc3a41e7a070551bdfdfb0b65b5314241152d8bd31f87c22da"
   end
 
-  # MSG_NOSIGNAL is only defined in Linux
-  # https://github.com/AlisterT/openjazz/pull/7
-  patch :DATA
-
   def install
-    # the libmodplug include paths in the source don't include the libmodplug directory
-    ENV.append_to_cflags "-I#{Formula["libmodplug"].opt_include}/libmodplug"
-
-    system "autoreconf", "-ivf"
-    system "./configure", "--prefix=#{prefix}",
-                          "--bindir=#{pkgshare}",
-                          "--disable-dependency-tracking"
-    system "make", "install"
-
-    # Default game lookup path is the OpenJazz binary's location
-    (bin/"OpenJazz").write <<~EOS
-      #!/bin/sh
-
-      exec "#{pkgshare}/OpenJazz" "$@"
```

Comment:
No idea if useful, but you could pass `-DDATADIR="#{pkgshare}/OpenJazz"` to cmake if needed.

Reply:
Hmmm.  I'm getting a CMake warning when I try that saying that `DATADIR` is a manually-specified valuable not used by the project (and then OpenJazz just keeps looking in the current directory by default after it's built).

Reply:
Oh right, it must be `DATAPATH`.

Reply:
Ah, yep.  That did it.